include/sdf/Collision.hh

```diff
@@ -75,6 +76,16 @@ namespace sdf
     /// \param[in] _name Name of the collision.
     public: void SetName(const std::string &_name);
 
+    /// \brief Get the density of the collision.
+    /// The density of the collision must be unique within the scope of a Link.
+    /// \return Density of the collision.
+    public: double Density() const;
+
+    /// \brief Set the density of the collision.
+    /// The density of the collision must be unique within the scope of a Link.
+    /// \param[in] _name Density of the collision.
```

Comment:
```suggestion
    /// \param[in] _density Density of the collision.
```

Reply:
Corrected in b092904db4ef5c491672619076e75f4509b3369b